aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -9,6 +9,7 @@ proposals:
     execution_mode: MultiStep
     update_sequence:
       - DefaultGasWithOverrideOld:
```

Comment:
Is it correct to use `DefaultGasWithOverrideOld` here? @zjma

Reply:
yes

Reply:
And, it's correct to use `DefaultGas` at the end of Step 2?

Reply:
correct!